core/selection.cpp

```diff
@@ -254,6 +254,17 @@ extern "C" void deselect_trip(struct dive_trip *trip)
 	}
 }
 
+extern "C" struct dive_trip *single_selected_trip()
+{
+	if (amount_trips_selected != 1)
+		return NULL;
+	for (int i = 0; i < trip_table.nr; ++i) {
+		if (trip_table.trips[i]->selected)
+			return trip_table.trips[i];
+	}
+	return NULL; // shouldn't happen
```

Comment:
print out a warning?

Reply:
done.

Reply:
it must be my day to be blind... I don't see it...

Reply:
Forgot "-a" in "commit --amend".